contents/careers/designer.md

```diff
@@ -8,37 +8,43 @@ showTitle: true
 
 PostHog exists to increase the number of successful products in the world.
 
-We're not just building software, our product is the company. This role is to shape how we get design into every part of that - from brand, to web presence, our engagement with our blossoming community, and into the platform itself.
+We're not just building software, our product is the company. We're building our first design team and this role is for a lead designer to shape how we shape our visual design across branding, website, documentation, social media, merch (!), and anywhere else you can see that needs attention.
+
+Whilst this isn't a UX role, we'd expect some feedback from a visual perspective on the product itself too, for example in the aesthetics of our components.
 
 ## Ambition and engineering have taken us a really long way, so far
 
 PostHog's engineering culture is best-in-class, and we've had explosive user growth from this alone.
 
 We [launched a four week old minimum viable product in February](/handbook/company/story), and since then have been deployed in thousands of places around the world.
 
-PostHog is used to help indie game designers make it more fun to defend earth from an alien threat. We are used by multinational organizations running software that powers everything from banks to airlines. We are used by startups disrupting their own industries.
+Our platform is used to help indie game designers make it more fun to defend earth from an alien threat. We are used by mulitnational organizations running software that powers everything from banks to airlines. We are used by startups disrupting their own industries.
```

Comment:
I know the content here is from a previous PR but this is confusing to me

Reply:
edited it - just trying to make point that we are used in loads of cool places already versus ie producing super dull software to help companies hire faster

Reply:
Much better, thanks!